static/download_data.html

```diff
@@ -87,6 +87,70 @@ <h2>AddBiomechanics 1.0 Dataset (High Quality Core Data):</h2>
         Project</a>
       <a href="assets/AddBiomechanics_Dataset_Paper.pdf" class="btn btn-secondary btn-lg">Read the Pre-Print</a>
     </p>
+    <style>
```

Comment:
I think it would be better to move this style section to the [custom.css](https://github.com/keenon/AddBiomechanics/blob/main/static/css/custom.css) file, which is already imported in this html.